extensions/amp-ad/0.1/amp-ad-cross-domain-iframe-handler.js

```diff
@@ -140,6 +146,7 @@ export class AmpAdApiHandler {
       return timerFor(this.baseInstance_.win).timeoutPromise(TIMEOUT_VALUE,
           this.adResponsePromise_,
           'timeout waiting for ad response').catch(e => {
+            this.timeout_ = true;
```

Comment:
Can we call this `timedOut`? `timeout` makes me think this is a `timeout = setTimeout(...)`, which it isn't.